<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Match</class>
 <widget class="QWidget" name="Match">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>750</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>750</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QFrame#frame {background-color: #302E2B}
QLabel {color: white;}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QFrame" name="frChessBoard">
    <property name="geometry">
     <rect>
      <x>75</x>
      <y>94</y>
      <width>560</width>
      <height>560</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QLabel" name="lbOpponentName">
    <property name="geometry">
     <rect>
      <x>75</x>
      <y>25</y>
      <width>500</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Opponent</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbPlayerName">
    <property name="geometry">
     <rect>
      <x>75</x>
      <y>663</y>
      <width>500</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>You</string>
    </property>
   </widget>
   <widget class="QFrame" name="frCapturedO">
    <property name="geometry">
     <rect>
      <x>75</x>
      <y>54</y>
      <width>560</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #F4DCC5;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QWidget" name="horizontalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>561</width>
       <height>31</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="hboxOpponent"/>
    </widget>
   </widget>
   <widget class="QFrame" name="frCapturedP">
    <property name="geometry">
     <rect>
      <x>75</x>
      <y>687</y>
      <width>560</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #F4DCC5;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QWidget" name="horizontalLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>561</width>
       <height>31</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="hboxPlayer"/>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>94</y>
      <width>225</width>
      <height>560</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
color: black;
background-color: #EF9A09;
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QTableWidget" name="tableWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>225</width>
       <height>500</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QTableWidget {background-color: rgba(0,0,0,0);
color:white;}
QHeaderView::section {
background-color: #302E2B;
color: white}</string>
     </property>
     <column>
      <property name="text">
       <string>Turn</string>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="background">
       <color alpha="0">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </property>
     </column>
     <column>
      <property name="text">
       <string>White</string>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="background">
       <color alpha="0">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Black</string>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="background">
       <color alpha="0">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </property>
     </column>
    </widget>
    <widget class="QPushButton" name="btnBack">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>520</y>
       <width>75</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Ubuntu</family>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{background-image: url(:/assets/back-button.png);}
QPushButton:hover{background-color: #FBC061}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="btnExit">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>520</y>
       <width>75</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Ubuntu</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Exit</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnForward">
     <property name="geometry">
      <rect>
       <x>75</x>
       <y>520</y>
       <width>75</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Ubuntu</family>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{background-image: url(:/assets/forward-button.png);}

QPushButton:hover{background-color: #FBC061}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
